spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -115,6 +149,23 @@
           hash_including(**analytics_args, success: true),
         )
       end
+
+      it 'adds costs' do
+        allow(controller).to receive(:load_async_state).and_return(async_state)
+        allow(async_state).to receive(:done?).and_return(true)
+        allow(async_state).to receive(:result).and_return(adjudicated_result)
```

Comment:
adjudicated result reflects result when a user goes through ipp. Stages in result appears to be the same whether or not the user's residential address matches their state id.

Reply:
Want to check if this is accurate re:Expected costs that should be tracked/added in db

**remote, aamva supported state:** 
threatmetrix cost x 1 (threatmetrix stage)
lexis nexis cost x 1 (resolution stage)
aamva cost x 1 (state id stage)

**remote, aamva unsupported state:** 
threatmetrix cost x 1 (threatmetrix stage)
lexis nexis cost x 1 (resolution stage)

**ipp, aamva supported state, same_address_as_id is true:** 
threatmetrix cost x 1 (threatmetrix stage)
lexis nexis cost x 2 (resolution stage, residential stage)
aamva cost x 1 (state id stage)

**ipp, aamva unsupported state, same_address_as_id is true:** 
threatmetrix cost x 1 (threatmetrix stage)
lexis nexis cost x 2 (resolution stage, residential stage)

**ipp, aamva supported state, same_address_as_id is false:** 
threatmetrix cost x 1 (threatmetrix stage)
lexis nexis cost x 2 (resolution stage, residential stage)
aamva cost x 1 (state id stage)

**ipp, aamva unsupported state, same_address_as_id is false:** 
threatmetrix cost x 1 (threatmetrix stage)
lexis nexis cost x 2 (resolution stage, residential stage)